.github/PULL_REQUEST_TEMPLATE/bugfix.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug fix
+about: Create a PR to improve this project
+title: ''
+labels: triage
+assignees: ''
+
+---
+
+**What issue is this PR fixing**
+Example:
+`closes #123`
+`fixes #456`
+
+**What is being changed**
+A clear description of what this PR brings.
+
+**Quality**
```

Comment:
All the items in the checklist below are already checked in our CI/CD flows. If the build doesn't work locally, it'll also fail in CI, and if it doesn't has unit or integration tests we also can add rules in the coverage tool in order to stop the PR. I understand that the checklist helps all to avoid low quality PRs to be created, but I personally don't see that as a problem, because the person will work to fix the issues in the PR detected during the CI verifications. I'd remove this checklist and improve the CI verifications instead.

Reply:
@mirceanis  Could you explain your point of view on this, please?

Reply:
It happens a lot that folks post a PR but don't check back to see if the CI scripts failed or if anyone is interested in reviewing the PR.

This checklist should help with that, since folks might be more aware of what will fail

Reply:
There is one more thing.

For first time contributors we have to manually run the workflows so it makes more sense to have these things already checked locally before waiting for CI/CD to do its thing.